src/components/pages/my-bookings/MyBookings.jsx

```diff
@@ -73,7 +73,7 @@ class MyBookings extends Component {
             exact
             path={'/reservations/:details(details)/:bookingId([A-Z0-9]+)/:qrcode(qrcode)'}
           >
-            <QrCodeContainer />
+            {!isQrCodeFeatureDisabled && <QrCodeContainer />}
```

Comment:
ça serait plus clair sans la double négation
```suggestion
            {isQrCodeFeatureActive && <QrCodeContainer />}
```

Reply:
Ainsi il faudrait remplacé le selector selectIsFeatureDisabled par selectIsFeatureActive

Reply:
Si j'inverse, cela me fait modifier plus de 10 composants

Reply:
Comme discuté, c'est utilisé à plusieurs endroits et ça impacte une dizaine de composants. Ces check de feature flipping sont destinés à être supprimés. On laisse en l'état